automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/UserFixtures.scala

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.dsde.firecloud.fixture
+
+import org.broadinstitute.dsde.firecloud.config.{AuthToken, Config, Credentials}
+import org.broadinstitute.dsde.firecloud.page.workspaces.WorkspaceListPage
+import org.broadinstitute.dsde.firecloud.test.{CleanUp, WebBrowserSpec}
+import org.openqa.selenium.WebDriver
+import org.scalatest.Suite
+
+trait UserFixtures extends CleanUp { self: WebBrowserSpec with Suite =>
+
+  def withSignIn(user: Credentials,
+                 cleanUp: Boolean = true)
+                (testCode: (WorkspaceListPage) => Any)(implicit webdriver: WebDriver): Unit = {
+
+    val listPage = signIn(user)
+    val userEmail: String = user.email
+
+    try {
+      testCode(listPage)
+    } finally {
+      if (cleanUp) {
+        try {
+          listPage.signOut()
```

Comment:
This signs them out of FireCloud but not signed out of Google. To sign out of Google you can navigate to: https://mail.google.com/mail/logout

Reply:
should we sign out of google in addition to firecloud, you think?